docs/communication-android.md

```diff
@@ -34,8 +34,8 @@ public class MainActivity extends ReactActivity {
       protected Bundle getLaunchOptions() {
         Bundle initialProperties = new Bundle();
         ArrayList<String> imageList = new ArrayList<String>(Arrays.asList(
-                "http://foo.com/bar1.png",
-                "http://foo.com/bar2.png"
+                "https://foo.com/bar1.png",
+                "https://foo.com/bar2.png"
```

Comment:
Let's revert this and all the `foo.com` URLs please

Reply:
(Context: foo.com has invalid SSL Certificate and will make sample snippets not work)

Reply:
Is there any _specific_ reason to use `foo.com`? Since they are giving `404`.
If not, we can use something more secure?

Reply:
Yup we can definitely update this

Reply:
Will look at it as soon as I get some time.

Reply:
https://dummyimage.com/600x400/ should do the job

Reply:
Sure :)